smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -813,7 +813,7 @@ def deviceHandler(evt) {
 		}
 	} else if (deviceInfo) {
 		if (deviceInfo.callbackUrl) {
-			sendToHarmony(evt, deviceInfo.callbackUrl)
+			sendToHarmonyCloud(evt, deviceInfo.callbackUrl)
```

Comment:
In line 810 it asks for state.harmonyHubs and sends data to sendToHarmony if there is any, so I thought this was trying to determine where to send the information locally or to the cloud. In that case maybe I should ask what is the URL for their cloud and if the callback contains their cloud url then I route it to sendToHarmonyCloud